scripts/handlers/divide.js

```diff
@@ -12,7 +12,7 @@ function divide(x, y) {
   if (typeof x !== 'number') { throw new TypeError('x'); }
   if (typeof y !== 'number') { throw new TypeError('y'); }
 
-  const result = _;
+  const result = y / x;
```

Comment:
can you check your debugger again, it should be like this `const  result = x/y`

Reply:
yes, you are right. But check the line 89 in divide.js. it says: divide the last result by the user's number and reassign lastResult
that's why i got confused